src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$cell-size: 100px;
+$default-month-length: 31;
+$days-of-week: (
+  'mon': '0',
+  'tue': '1',
+  'wed': '2',
+  'thu': '3',
+  'fri': '4',
+  'sat': '5',
+  'sun': '6',
+);
+$start-day-index: 7;
```

Comment:
The `$start-day-index` should be between 0 and 6, inclusive, to represent the index of the first day of the week. The current value 7 is outside of this range, which could lead to incorrect calculations or representations in the styles.